modules/nf-core/star/starsolo/meta.yml

```diff
@@ -0,0 +1,73 @@
+name: "starsolo"
+description: write your description here
+keywords:
+  - align
+  - count
+  - genome
+  - reference
+tools:
+  - "starsolo":
+      description: "Mapping, demultiplexing and quantification for single cell RNA-seq."
+      homepage: "https://github.com/alexdobin/STAR/"
+      documentation: "https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md"
+      doi: "10.1101/2021.05.05.442755"
+      licence: ["MIT"]
+input:
```

Comment:
Shouldn't the `index` from the input-section in `main.nf` be described here?

Reply:
Done!